fab/config.example.py

```diff
@@ -0,0 +1 @@
+GITHUB_APIKEY = ''
```

Comment:
Relatedly, I found that enabling only `public_repo` on https://github.com/settings/tokens/new was sufficient to deploy.